en/docs/ipaas/micro-integrator/develop-integrations-with-integration-studio.md

```diff
@@ -67,6 +94,29 @@ The Choreo DevOps Portal allows for greater flexibility in defining configuratio
 
 For a comprehensive list of parameters that can be configured as environment variables, please refer to the [WSO2 API Manager Documentation - Injecting Parameters - Supported parameters](https://apim.docs.wso2.com/en/latest/integrate/develop/injecting-parameters/#supported-parameters).
 
+## Configuring logging
+
+Proper logging mechanisms are crucial when developing and maintaining a software system. Logging captures events and messages that occur during an application's execution, providing valuable information for troubleshooting and debugging purposes.
+
+In Micro Integrator (MI) instances, logging can be configured and customized to meet specific requirements. The logging
```

Comment:
Suggestion:
In Micro Integrator instances, you can configure and customize logging depending on your requirement. You can add logging configurations to each MI instance's environment, which allows you to fine-tune logging depending on the specific environment or deployment scenario.

To configure logging in MI instances, you can use environment variables with specific naming conventions as follows:
1. Start the variable name with `logging_level_` followed by the package or class name.
2. Replace the dot character in the package name with an underscore.
3. Set the variable value to the required logging level for the corresponding package or class.

For example, to set the logging level for the `org.apache.synapse.transport.http.wire` package to `debug`, define the following environment variable:

```
export logging_level_org_apache_synapse_transport_http_wire=debug
```

This approach allows you to control the logging granularity, which helps you troubleshoot and identify issues faster.

Reply:
ack